Add disclaimer about using antivirus/antimalware solution (#1995)
epiphany-platform/epiphany: docs/home/SECURITY.md

You are strongly advised use encrypted over unencrypted communication between Epiphany components where possible. Please consider
this during planning your Epiphany deployment and configuration.

> We strongly advise to change default passwords wherever Epiphany configuration let you do so.

Reviewer: ```suggestion
We strongly advise to change all the default passwords in Epiphany configuration.
```

> We strongly advise to change default passwords wherever Epiphany configuration let you to do this.

Reviewer 2: ```suggestion
We strongly advise to change default passwords wherever Epiphany configuration lets you do so.
```

> We strongly advise to use antivirus/antimalware software wherever possible to prevent risks. Please consider this during planning

Reviewer 2: ```suggestion
We strongly advise to use antivirus/antimalware software wherever possible to prevent security risks. Please consider this during planning
```

> your Epiphany deployment and test if Epiphany components are installing correctly with making necessary changes in settings of your

Reviewer 2: ```suggestion
your Epiphany deployment and test if Epiphany components are installing correctly with necessary changes made in settings of your
```

> We strongly advise to use antivirus/antimalware software wherever possible to prevent security risks. Please consider this during planning
> your Epiphany deployment and test if Epiphany components are installing correctly with necessary changes made in settings of your
> antivirus/antimalware solution.

Reviewer: ```suggestion
We strongly advise to use antivirus/antimalware software wherever possible to prevent security risks.
Please consider this during planning of your Epiphany deployment. The configuration of security software may need to be evaluated and adjusted in order to not block Epiphany components from working properly.
```
